utils/users.js

```diff
@@ -149,4 +157,5 @@ module.exports = {
   getLowestLevelSkill,
   getPaginationLink,
   getUsernamesFromPRs,
+  checkRoleToUpdate,
```

Comment:
Is this unit tested?

Reply:
No there is no unit test for this because I didn't know for utils file we have to write unit test

Reply:
Done with changes.